releasenotes/notes/46693.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: networking
+issue:
+  - 46680
+
+releaseNotes:
+  - |
+    **Fixed** an issue where istio validation required address to be set on endpoints even if network is set.
```

Comment:
I don't know if we should write this as bug-fix. I believe it's releasing along with the WorkloadEntry update, so this is net new behavior.

Reply:
https://github.com/istio/istio/blob/d8b21e02e754b6e93bf259ff7e226fafaa9c16d3/releasenotes/notes/45150.yaml

Maybe we can make it just like this one.

Reply:
or perhaps we should skip it if they will be in the same release notes?

Reply:
+1 to Steven's suggestion